frontend/src/components/module-viewer/content/blocks/column/ColumnBlock.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Grid } from "@chakra-ui/react";
+
+import {
+  ContentBlockProps,
+  ColumnBlockState,
+} from "../../../../../types/ContentBlockTypes";
+import ColumnContent from "./ColumnContent";
+
+const ColumnBlock = ({
+  block: { content, id },
+  index,
+  editable,
+}: ContentBlockProps<ColumnBlockState>): React.ReactElement => {
+  if (index === undefined) {
```

Comment:
Would this ever happen? TypeScript should be enforcing that this is not possible...

Reply:
Left over code from a type change, I'll remove it